src/LandingPage/LandingPage.js

```diff
@@ -15,20 +15,10 @@ export default function LandingPage() {
                 buttonLabel="Register Now"
                 buttonLink="/login"
             />
-            <HeroBanner 
-                title="About Us" 
-                image={about} 
-                imageAlt="placeholder text"
-                buttonLabel="Read More"
-                buttonLink="/about"
-            />
-            <HeroBanner 
-                title="Our Goal" 
-                image={ourGoal} 
-                imageAlt="placeholder text"
-                buttonLabel="Read More"
-                buttonLink="/our-goal"
-            />
+            <div className="container">
+                <AboutUs />
```

Comment:
User currently does not see OurGoal animations on pageload.
Position OurGoal before About, or set animations to trigger once user has scrolled down

Reply:
Fair point. 
WIll run past the team on the call.